web/template/manage-restrictions.gohtml

```diff
@@ -23,48 +23,76 @@
-                                <input class="govuk-radios__input" id="f-severanceActionNotRequired" name="severanceAction" type="radio" value="severance-application-required" {{ if eq "severance-application-required" .SeveranceAction }}checked{{ end }}>
-                                <label class="govuk-label govuk-radios__label" for="f-severanceActionNotRequired">
-                                    Severance application is required
-                                </label>
+                            <div class="govuk-radios" data-module="govuk-radios">
+                                <div class="govuk-radios__item">
+                                    <input class="govuk-radios__input" id="f-severanceActionNotRequired" name="severanceAction" type="radio" value="severance-application-required" {{ if eq "severance-application-required" .SeveranceAction }}checked{{ end }}>
+                                    <label class="govuk-label govuk-radios__label" for="f-severanceActionNotRequired">
+                                        Severance application is required
+                                    </label>
+                                </div>
                             </div>
-                        </div>
-                    </fieldset>
+                        </fieldset>
 
-                    {{ else }}
-                    <fieldset class="govuk-fieldset">
-                        <legend class="govuk-fieldset__legend"><strong>Select an option:</strong></legend>
-                        {{ template "errors" .Error.Field.severanceAction }}
+                    {{ else if and (eq .CaseSummary.DigitalLpa.SiriusData.Application.SeveranceStatus "REQUIRED") (eq .FormAction "") }}
+                        <fieldset class="govuk-fieldset">
+                            <legend class="govuk-fieldset__legend"><strong>Select an option:</strong></legend>
+                            {{ template "errors" .Error.Field.donorConsentAction }}
 
-                        <div class="govuk-radios" data-module="govuk-radios">
-                            <div class="govuk-radios__item">
-                                <input class="govuk-radios__input" id="f-severanceActionRequired" name="severanceAction" type="radio" value="severance-application-not-required" {{ if eq "severance-application-not-required" .SeveranceAction }}checked{{ end }}>
-                                <label class="govuk-label govuk-radios__label" for="f-severanceActionRequired">
-                                    Severance application is not required
-                                </label>
+                            <div class="govuk-radios" data-module="govuk-radios">
+                                <div class="govuk-radios__item">
+                                    <input class="govuk-radios__input" id="f-donorConsentGiven" name="severanceAction" type="radio" value="donor-consent-given" {{ if eq "donor-consent-given" .SeveranceAction }}checked{{ end }}>
```

Comment:
I think it would be easier to name this field `donorConsentGiven` rather than `severanceAction` so the back-end knows clearly what it's handling